osu.Game/Screens/Menu/IdleTracker.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics;
+using osu.Framework.Input;
+using osu.Framework.Input.Bindings;
+using osu.Framework.Input.EventArgs;
+using osu.Framework.Input.States;
+
+namespace osu.Game.Screens.Menu
```

Comment:
WH yis this under the `Screens.Menu` namespace if it's used at a `Game` level?

Reply:
Because it is used only in the main menu. Looking at the folder structure, I guess it should be moved to the `osu.Game.Input` namespace.